ad-insertion/video-analytics-service/app/modules/GStreamerPipeline.py

```diff
@@ -188,25 +193,33 @@ def bus_call(bus, message, self):
             if self.state is "RUNNING":
                 logger.debug("Setting Pipeline {id} State to COMPLETED".format(id=self.id))
                 self.state = "COMPLETED"
-
             self.stop_time = time.time()
+            time.sleep(1)
```

Comment:
is the sleep required? maybe a comment on why it is needed?

Reply:
did some testing and do not need this. removed.